components/bl-webcamPhoto-component/README.md

```diff
@@ -1,40 +1,117 @@
 # Webcam Photo
 
-WebcamPhoto is a component of Backendless UI-Builder designer. Allows you to take a picture using a webcam on both
-mobile devices and desktops.
-
-## Usage
-
-### Styles and Settings
-
-<dl>
-<dt>Upload button label</dt>
-<dd>Label on the Upload button. Defaults to "Upload".</dd>
-<dt>Make photo button label</dt>
-<dd>Label on the Make photo button. Defaults to "Make photo".</dd>
-<dt>Snapshot button label</dt>
-<dd>Label on the Make snapshot button. Defaults to "Make snapshot".</dd>
-<dt>Done button label</dt>
-<dd>Label on the Done button. Defaults to "Done".</dd>
-<dt>Default disabled</dt>
-<dd>Indicates if the button is disabled.</dd>
-<dt>Popup link label</dt>
-<dd>Link label in popup. Defaults to "Info...".</dd>
-<dt>No yet  permission popup title</dt>
-<dd>Title in popup when the user has not yet allowed the use of the camera.</dd>
-<dt>No yet  permission popup description</dt>
-<dd>Description in popup when the user has not yet allowed the use of the camera.</dd>
-<dt>No permission popup title</dt>
-<dd>Title in popup when the user did not allow the use of the camera.</dd>
-<dt>No permission popup description</dt>
-<dd>Description in popup when the user did not allow the use of the camera.</dd>
-</dl>
-
-### Event Handlers and Bindable Properties
-
-<dl>
-<dt>On Save Image Event</dt>
-<dd>Triggered when the user clicks the "Done" button. </dd>
-<dt>Button Disabled Logic</dt>
-<dd>Used to change the value of the disabled field for a button.</dd>
-</dl>
+WebcamPhoto is a component of Backendless [UI-Builder](https://backendless.com/developers/#ui-builder) designer.
+The component allows taking a picture using a webcam on both mobile devices and desktops.
+
+<p align="center">
+  ![main thumbnail](./thumbnail.png)
+</p>
+
+## Properties
+
+| Property                        | Type       | Default value                                                                                      | Logic                 | Data Binding | UI Setting | Description                                                                           |
+|---------------------------------|------------|----------------------------------------------------------------------------------------------------|-----------------------|--------------|------------|---------------------------------------------------------------------------------------|
+| uploadButtonLabel               | *text*     | Upload                                                                                             |                       | NO           | YES        | controls label on the Upload button.                                                  |                                                                                                        |
```

Comment:
Upload => "Upload" - because it string
true => `true` - as it boolean

And so on

https://github.com/Backendless/ui-builder-library/wiki/PR-Checklist#readme